maxim/tests/test_langgraph.py

```
@@ -139,47 +134,71 @@ class GraphConfig(TypedDict):
 
 class TestLangGraph(unittest.TestCase):
     def setUp(self) -> None:
+        # This is a hack to ensure that the Maxim instance is not cached
+        if hasattr(Maxim, "_instance"):
+            delattr(Maxim, "_instance")
 
+        self.mock_writer = inject_mock_writer(logger)
         return super().setUp()
 
+    @langgraph_agent(name="ask_agent", logger=logger)
+    async def ask_agent(self, query: str) -> str:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)